specs/java.xml.ws/pom.xml

```diff
@@ -30,6 +30,10 @@
     <artifactId>org.apache.karaf.specs.java.xml.ws</artifactId>
     <name>Apache Karaf :: Specs :: Java Xml WS</name>
 
+    <properties>
+        <maven.compiler.target>8</maven.compiler.target>
```

Comment:
Why is this (and the following) targeting 8 rather than 11?

Reply:
Some specs still need Java 8 but specs gonna removed anyway.